mathesar/tests/api/test_data_file_api.py

```diff
@@ -124,6 +124,18 @@ def test_data_file_create_json(client, patents_json_filepath, header):
     )
 
 
+@pytest.mark.parametrize('header', [True, False])
+def test_data_file_create_excel(client, patents_excel_filepath, header):
+    num_data_files = DataFile.objects.count()
+
+    with open(patents_excel_filepath, 'rb') as excel_file:
+        data = {'file': excel_file, 'header': header}
```

Comment:
`header` is too vague. Use `has_header` or something along those lines. That will make it clear that this is a boolean flag.

Reply:
Updated it now. 👍🏻